### PR TITLE
Add google directions

### PR DESCRIPTION
**Pull request by vroland1**
linked the address from location details to open google directions, adjusted the CSS for the infobox
google_direction_real_value

issue #29

### CODE REVIEW

File: src/renderer/containers/map/infoBox/index.jsx [hunk 1]
```diff
@@ -32,8 +32,17 @@ export default class InfoBox extends React.Component {
       <div className="info-box">
         <div className="content">
           <h2>{details.name}</h2>
-          <p>{details.address1}</p>
-          <p>{details.address2}</p>
+           <a
+            href={`https://www.google.com/maps/place/${replace(
```
Comment (mihiramin89) [inline src/renderer/containers/map/infoBox/index.jsx]: can we move this href to a const at the top of the render? like this.

```
const conversion = addr => replace(/ /g, '+', addr);
const address = 'https://www.google.com/maps/place/${conversion(details.address1)}+${conversion(details.address2}+${details.zipCode}'


....
<a href={address} ...>
```

**Comment by mihiramin89** [comment]
Great job Tory! This works great!

File: src/renderer/containers/map/infoBox/index.jsx (hunk 1 above)
Comment (vroland1) [reply src/renderer/containers/map/infoBox/index.jsx]: Sure!

File: src/renderer/containers/map/infoBox/index.jsx [hunk 2]
```diff
@@ -32,8 +32,17 @@ export default class InfoBox extends React.Component {
       <div className="info-box">
         <div className="content">
           <h2>{details.name}</h2>
-          <p>{details.address1}</p>
-          <p>{details.address2}</p>
+           <a
```
[the remark this answers is not in the document]
Comment (vroland1) [reply src/renderer/containers/map/infoBox/index.jsx]: the formatting in the infobox looks fine, but now that I am running it with the actual data it seems like for the majority of locations that have an "address 2" value, it isn't usually something that would be included in a formal address or something that google maps would understand (i.e. "THIRD FLOOR" "USE SIDE ENTRANCE"). going to take it out of the href const!

google_direction_real_value

[the remark this answers is not in the document]
Comment (vroland1) [reply src/renderer/containers/map/infoBox/index.jsx]: not sure how I pulled in those changes from melissa and working on figuring out how to revert

**Pull request merged**